x/emissions/keeper/inference_synthesis/network_losses.go

```diff
@@ -8,66 +8,59 @@ import (
 	emissions "github.com/allora-network/allora-chain/x/emissions/types"
 )
 
-type WorkerRunningWeightedLoss struct {
-	SumWeight Weight
-	Loss      Loss
+type RunningWeightedLoss struct {
+	UnnormalizedWeightedLoss Loss
+	SumWeight                Weight
 }
 
-// Update the running weighted loss for the worker
-// Source: "Weighted mean" section of: https://fanf2.user.srcf.net/hermes/doc/antiforgery/stats.pdf
-// nextValue format - raw value
-// weight format - logged value
+// Update the running information needed to calculate weighted loss per worker
```

Comment:
I think it would be helpful to have a comment description of what this function does mathematically (is this going to be defined in the white paper?)

Reply:
it's just taking numerator and denominator needed to calculate a simple weighted average. I'll make the comment clearer.